### DISCUSSION TITLE
Way to change fonts dynamically

### DISCUSSION THREAD

**Discussion by gabriel-fsa** [jsvine/pdfplumber#792] [Ideas]
I'm using pdfplumber to generate images for training a tesseract model.
And I'm looking for a way to change the font of the entire PDF (Arial to Times New Roman, for example) before converting the PDF to an image. I've looked for this functionality both in PDFplumber and in Pdfminer.six and I didn't find it. Would this functionality be useful for the project?

**Comment by jsvine** [comment]
Unfortunately for your needs, `pdfplumber` does not have PDF-writing abilities. You might instead be interested in [`pikepdf`](https://github.com/pikepdf/pikepdf) or other PDF-creation tools listed [here](https://johannesfilter.com/python-and-pdf-a-review-of-existing-tools/).
